book/01-introduction/sections/installing.asc

```diff
@@ -29,14 +29,14 @@ Auf einem Debian-basierten System, wie Ubuntu, steht `apt` zur Verfügung:
 $ sudo apt install git-all
 ----
 
-Auf der Git Homepage http://git-scm.com/download/linux[] findet man weitere Möglichkeiten und Optionen, wie man Git unter einem Unix-basierten Betriebssystem installieren kann.
+Auf der Git-Homepage http://git-scm.com/download/linux[] findet man weitere Möglichkeiten und Optionen, wie man Git unter einem Unix-basierten Betriebssystem installieren kann.
 
 ==== Installation unter macOS
 
 (((macOS, Installation)))
 Es gibt mehrere Möglichkeiten, Git auf einem Mac zu installieren.
 Am einfachsten ist es wahrscheinlich, die Xcode Command Line Tools zu installieren.(((Xcode)))
-Bei Mavericks (10.9) oder neueren Versionen kann man dazu einfach 'git' im Terminal eingeben.
+Bei Mavericks (10.9) oder neueren Versionen kann man dazu einfach `git` im Terminal eingeben.
```

Comment:
Inhaltlich bin ich mit deiner Korrektur einverstanden.
Aber, ich weiß, dass der Buchautor sehr großen Wert darauf legt, dass seine Textformatierungen erhalten werden und die ist im Originaltext wie vor.
Ich bitte dich, im englischen [Repo](https://github.com/progit/progit2/blob/master/book/01-introduction/sections/installing.asc) dazu einen PR zu starten. Wenn er abgelehnt werden sollte, müsste deine Korrektur hier wieder rückgängig gemacht werden.

Reply:
Ich wollte mich eigentlich darauf konzentrieren das Buch zu lesen. Wenn ich schon Fehler finde, dann reporte ich diese gerne, weil sie praktisch "umsonst" gefunden werden. Ob diese dann angenommen oder abgelehnt werden, entscheidet dann ihr.

Sprachliches können wir -- wie oben -- gerne noch diskutieren, aber jetzt auch noch im englischen Repo einen PR zu erstellen möchte ich eigentlich nicht. Deshalb mein Gegenvorschlag: Stelle du doch den PR und nehme ihn hier später wieder zurück, sollte er dort abgelehnt werden. Einverstanden?

Reply:
Das hat nichts mit "Abschieben von Aufwand" zu tun. 
Ich weiß, dass @ben großen Wert darauf legt, dass er "Entdecker" von Typos oder anderer Fehler belohnen will, indem sie in der History der Commits erscheinen.

Reply:
Auf diese Belohnung lege ich großen Wert. Man kann Leute doch als "Co-Author" hinzufügen. Das sollte völlig reichen. Falls das nicht geht, kann ins Log geschrieben werden, dass ich den "Fehler" gefunden habe. Und falls das nicht gewünscht ist, darfst du gerne die Lorbeeren ernten.

Ich verzichte darauf, weil ja noch mehr davon kommen könnten, wenn ich wieder zum Lesen des Buches komme. Was weitere PRs zur Folge hätte ... Aber wie schon geschrieben, liegt mein Hauptaugenmerk für den Moment auf dem **Lesen** des Buches und nicht, dies zu korrigieren.

Reply:
Soll ich den PR im englischen Repo erstellen? Würde dich auch als Entdecker nennen, @mo-gul

Reply:
@cbachhuber, gerne.

Reply:
PR ist offen: https://github.com/progit/progit2/pull/1446

Reply:
Merged. Damit können wir diesen Satz so lassen 👍